actors/builtin/power/power_state.go

```diff
@@ -71,13 +87,13 @@ func (st *State) minerNominalPowerMeetsConsensusMinimum(s adt.Store, minerPower
 	}
 
 	var minerSizes []abi.StoragePower
-	var pwr abi.StoragePower
-	if err := adt.AsMap(s, st.ClaimedPower).ForEach(&pwr, func(k string) error {
+	var claimed Claim
```

Comment:
this method isnt used at all, and if it were, i would be throwing red flags at it from 50 feet away. Iterating the miner set during any onchain operation is asking for pain.

Can we just delete this one now?

Reply:
As discussed, the conditions higher up in this function mean this iteration happens only very early on in the network life, when no miner yet meets the size threshold.

This method *should* be called somewhere, as part of validating a miner is eligible for EPost. Maybe that call should come from off-chain.